src/utils/hooks.tsx

```diff
@@ -53,3 +60,28 @@ export function useViewContext(): any {
 
   return { setViewContext, ...canvasProps };
 }
+
+/**
+ * useAnimationFrame
+ */
+/* eslint-disable-next-line @typescript-eslint/no-explicit-any */
+export const useAnimationFrame = (callback, dependencies?: any[]): void => {
+  // Use useRef for mutable variables that we want to persist
+  // without triggering a re-render on their change
+  const requestRef = useRef<number>();
+
+  function animate(): void {
+    callback();
+    requestRef.current = requestAnimationFrame(animate);
+  }
+
+  useEffect(() => {
+    requestRef.current = requestAnimationFrame(animate);
```

Comment:
```suggestion
    animate();
```

This was a bug that caused the other jitter on local run.